### ISSUE TITLE
PvP flag

### ISSUE THREAD

**Issue by Vasharij** [Valhalla-Gaming/Tracker#4353]
I was flying over Stormpeaks with my 85 hunter horde, I had just tamed Skoll and thought to go for a stroll and see if the time lost proto drake was there. While flying I met somebody that started following me, I landed near Frosthold and he attacked me, a level 100 priest. Now, I hate PvP and I have it always disabled. Why was he able to attack me? He said he was following me on purpose, so I was flagged even before reaching Frosthold. What happened?

**Comment by Aveta** [comment]
Did you accidentally fly over Winterspring before getting to SP? If not, then no idea.

**Comment by Vasharij** [comment]
Yes, I went from Sholazar Basin to Stormpeaks, through Winterspring. But it should remove it after flying over there. Shouldn't it?

**Comment by Aveta** [comment]
Never noticed it being removed after leaving WS, idk if it's intended or not. So yeah, that would be the reason : /

**Comment by Vasharij** [comment]
Neither did I. This is why I thought I wasn't flagged for pvp, infact I didn't understood what was happening when he attacked me.
It never happened to me in retail, not even on other private server.

**Comment by Tregeta** [comment]
I do believe that it's only when wintergrasp battle is actually going on that it flags you/teleports you out. If the battle isn't going on it's like a normal zone. I will need to confirm this a bit later on tonight however to be certain.
